internal/database/migrations/00001_initial_migration.sql

```diff
@@ -78,6 +77,39 @@ CREATE INDEX idx_memberships_bucket_id ON memberships (bucket_id);
 -- Partial unique index for active memberships only (soft-delete aware)
 CREATE UNIQUE INDEX idx_memberships_user_bucket ON memberships (user_id, bucket_id) WHERE deleted_at IS NULL;
 
+-- Folders table
+CREATE TABLE folders
```

Comment:
May be out of scope but we need to go through this file and replace VARCHAR(n) to VARCHAR or TEXT